src/metabase/db/custom_migrations/pulse_to_notification.clj

```diff
@@ -135,15 +147,24 @@
                                      {:channel_type "channel/http"
                                       :channel_id    (:channel_id pc)})))
                                 pcs)]
+         (qs/delete-trigger scheduler (send-pulse-trigger-key pulse-id pc))
```

Comment:
Note: Don't strictly need this, since they'll already have been deleted, but doesn't do any harm, and our tests are coupled to it.